tests/AnalyzeSpec.hs

```diff
@@ -1763,6 +1763,76 @@ spec = describe "analyze" $ do
               |]
         in expectPass code $ Valid $ sNot Abort'
 
+  describe "chain-data" $ do
+    describe "block-height field" $
+      let code =
+            [text|
+              (defun test:integer ()
+                @model [(property (>= result 0))]
+                (at "block-height" (chain-data)))
+            |]
+      in expectVerified code
+
+    describe "block-time field" $
+      let code =
+            [text|
+              (defun test:bool ()
+                (enforce
+                  (= (at "block-time" (chain-data))
+                     (time "2024-03-21T00:00:00Z"))
+                  "block time can't be this arbitrary date"))
```

Comment:
"can be"?

Reply:
The string is the error message that is displayed if the `enforce` fails, so this is the message we would see when the unexpected happens.